client/src/holdings/HoldingsList.tsx

```diff
@@ -62,9 +62,10 @@ export default function HoldingsList({user}: HoldingsListProps) {
     setOpen([...open]);
```

Comment:
Commit title should be

[Client] Sort Names on adding Holding (fixes #7)

The format of (fixes #7) is important so that github can automatically pick up that this fixes that bug.